src/frontend/src/components/BulletList.tsx

```diff
@@ -30,9 +32,12 @@ const BulletList: React.FC<BulletListProps> = ({ title, headerRight, list, order
     builtList = <ol style={styles.bulletList}>{listPrepared}</ol>;
   }
   return (
-    <PageBlock title={title} headerRight={headerRight} defaultClosed={defaultClosed}>
+    <>
+      <Typography variant="h3" sx={{ fontFamily: 'Oswald,sans-serif' }}>
```

Comment:
I don't think you need this fontFamilty stuff, typography on its own should set the font correctly